website/util/sanitize.py

```diff
@@ -99,6 +99,19 @@ def safe_unescape_html(value):
     return value
 
 
+# TODO: Remove safe_unescape_html when mako html safe comes in
+def safe_unescape_html_in_results(func):
```

Comment:
What is the advantage of this decorator? It seems like it reduces clarity, and doesn't save a lot of boilerplate.

(In fact, the name you have to type is longer than if you'd called `safe_unescape_html` directly)